README.md

```diff
@@ -9,25 +9,54 @@ go-libp2p-conn
 
 > A library providing 'Connection' objects for libp2p.
 
+This package offers wrappers for `go-libp2p-transport` raw types,
+exposing `go-libp2p-interface-conn` types.
+
+It negotiates either plaintext or secio over the raw connection
+using `go-multistream`.
 
 ## Table of Contents
 
 - [Install](#install)
+- [Usage](#usage)
+- [Protocol overview](#protocol-overview)
 - [Contribute](#contribute)
 - [License](#license)
 
 ## Install
 
 ```sh
-make install
+go get -u github.com/libp2p/go-libp2p-conn
```

Comment:
Please leave this as `make install`. We use `gx` to manage our dependencies to prevent breaking changes in dependencies from breaking dependent packages.

Reply:
There is no `make install` target in the Makefile.

Reply:
Ah. Good point. That should be `make deps`.

Reply:
Done.